pkg/webhook/alias.go

```diff
@@ -70,4 +85,13 @@ var (
 
 	// Errored indicates that an error occurred in the admission request.
 	Errored = admission.Errored
+
+	// Authenticated indicates that the token review should be allowed for the given reason.
+	Authenticated = authentication.Authenticated
+
+	// Unauthenticated indicates that the token review should be unauthorized for the given reson.
+	Unauthenticated = authentication.Unauthenticated
+
+	// AuthenticationErrored indicates that an error occurred in the token review
+	AuthenticationErrored = authentication.Errored
```

Comment:
Not about: `Errored = admission.Errored` given the expanded scope would we want to make this `AdmissionErrored`? I hadn't given this would be a breaking change.